src/core/worker.c

```diff
@@ -621,12 +620,11 @@ QuicWorkerLoopCleanup(
 //
 BOOLEAN
 QuicWorkerLoop(
-    _Inout_ struct CXPLAT_EXECUTION_CONTEXT* Context,
-    _Inout_ uint64_t* TimeNow,
-    _In_ CXPLAT_THREAD_ID ThreadID
+    _Inout_ void* Context,
+    _Inout_ CXPLAT_EXECUTION_STATE* State
     )
 {
-    QUIC_WORKER* Worker = (QUIC_WORKER*)Context->Context;
+    QUIC_WORKER* Worker = (QUIC_WORKER*)Context;
```

Comment:
This is not `CXPLAT_EXECUTION_CONTEXT*` ?

Reply:
It's both.